### ISSUE TITLE
Not able to change edge label font properties.

### ISSUE THREAD

**Issue by dheerajbapat** [almende/vis#3490]
Changing font face of a particular edge label is not working.
I tried to updated font face and font color. 
```
edge.options.font.face = 'arial';
edge.options.font.color = '#ff0000';
```

However changing dashes (or some other property) is working.
`edge.options.dashes = true;`

**Comment by wimrijnders** [comment]
Duplicate of #3408.
There is a fix on the way for this and similar: #3486. Hold on, almost there!

**Comment by dheerajbapat** [comment]
Thanks! waiting for next release :smiley: 
Hope it will fix same issue with changing edge.options.arrows as well. :crossed_fingers:

**Comment by wimrijnders** [comment]
> Hope it will fix same issue with changing edge.options.arrows as well.

Is there an issue for that? Refresh my memory, please.

**Comment by dheerajbapat** [comment]
Yes, changing font face or color for particular edge label is not taking effect.
For example in global options I say edge label font face courier new and dynamically if I try to change some labels to webdings or arial, this is not working.

May I know possible date for next release?

**Comment by wimrijnders** [comment]
The fix for that is #3486. There's no date fixed yet for a next release. But it should be soon, seeing that there are a lot of fixes pending.

**Comment by wimrijnders** [comment]
@dheerajbapat `vis.js v4.21.0` has just been released. Would you mind checking if you still have the problems with changing font face or color for edges?

**Comment by dheerajbapat** [comment]
I upgraded to v4.21.0 and checked. Unfortunately issue still exists. 
Please let me know if you need any further clarification for the issue.

**Comment by xwenyuan** [comment]
excuse me , here i have a similar issue with edit edges.
for example,in this demo:   **http://visjs.org/examples/network/other/manipulationEditEdgeNoDrag.html**
we can change label in edit_edge mode, now i change the codes and then run , giving one of the edges properties "{arrows:from; label:hello}", the edge will have an arrow and label named 'hello'.
now ,when i again edit the same edge,in the form ,i get the history data with label filled in 'hello',but the arrows was filled in 'undefined'.
it seems only label will work properly.
